pkg/filesystem/local_directory_windows.go

```diff
@@ -606,7 +607,7 @@ func (d *localDirectory) RemoveAllChildren() error {
 			err = subdirectory.RemoveAllChildren()
 			subdirectory.Close()
 			if err != nil {
-				return err
+				return util.StatusWrapf(err, "%s: ", name)
```

Comment:
Using `util.StatusWrap()` recursively might be a bit wacky, because it means you end up getting error messages along these lines:
```
src: myproject: mylibrary: mypackage: Failed to remove bob.c
```
Maybe the better thing to do is to make use of pkg/filesystem/path's Trace type here?
```go
func (d *localDirectory) RemoveAllChildren() error {
	return d.removeAllChildren(nil)
}

func (d *localDirectory) RemoveAllChildren(dPath *path.Trace) error {
	names, err := readdirnames(d.handle)
	if err != nil {
		return util.StatusWrapf(err, "Failed to read contents of directory %#v", dPath.String())
	}
	...
	for _, name := range names {
		childPath := dPath.Append(name)
		...
		err = subdirectory.removeAllChildren(childPath)
		...
```

Reply:
I think I did this, but it should not be part of this PR. This is just meant to do some simple housekeeping stuff. So I can merge a couple of commits.